Makefile

```diff
@@ -32,3 +34,18 @@ release: b2d_iso get_gh-release
 	cp $(BUILD_DIR)/minishift-b2d.iso release/
 	$(BUILD_DIR)/gh-release checksums sha256
 	$(BUILD_DIR)/gh-release create minishift/minishift-b2d-iso $(VERSION) master v$(VERSION)
+
+.PHONY: prepare_for_test
+prepare_for_test: DOCKER_MACHINE_PATH=$(BIN_DIR)/docker-machine
+prepare_for_test:
+	@if [ ! -x $(DOCKER_MACHINE_PATH) ]; then \
```

Comment:
This if statement is redundant, provided you make this a non phony target. Let's use the tools we use properly.